File: Java/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printCharacteristics() {
        System.out.printf("license: %s , Driver: %s \n", license, driver.name);
    }
}

File: PHP/Car.php
<?php
require_once('account.php');
class Car{
    public $id;
    public $license;
    public $driver;
    public $passengers;

    public function __construct($license,$driver){
        $this -> license = $license;
        $this -> driver = $driver; 
    }
    public function printCharacteristics(){
        echo "license: $this->license, driver: {$this->driver->name}, password: {$this->driver->password}";
    
    } 
}